pkg/controlplane/controller/leaderelection/leaderelection_controller_test.go

```diff
@@ -683,6 +682,10 @@ func TestController(t *testing.T) {
 
 	for _, tc := range cases {
 		t.Run(tc.name, func(t *testing.T) {
```

Comment:
For the reviewer: I have tested this with `time.Sleep(time.Second*6)` here to expire all `time.Now()` from above in the fixtures.